mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -42,7 +42,11 @@ def __init__(self):
 
     def append(self, payload):
         """! Append stream buffer with payload and process. Returns non-KV strings"""
-        self.buff += payload.decode('utf-8')
+        logger = HtrunLogger('CONN')
+        try:
+            self.buff += payload.decode('utf-8')
+        except UnicodeDecodeError:
+            logger.prn_err("UnicodeDecodeError encountered!")
```

Comment:
Total nit pick here: I'd go with a `prn_wrn` here since the `htrun` will continue to run.

Reply:
logger.prn_err seemed like an obvious choice at first, but then looking at other places where it is used, we do return/raise an exception when we log the message with prn_err, but here we won't be doing anything like that, so I see your point. I will change that.